common.gypi

```diff
@@ -11,7 +11,7 @@
     'configurations': {
       'Debug': {
         'defines': [ 'DEBUG', '_DEBUG' ],
-        'cflags': [ '-g', '-O0', '-fwrapv' ],
+        'cflags': [ '-g' ],
```

Comment:
The options "-O0 -fwrapv" are not valid for the zOS compiler.
So I took those out and created the conditional below in the file.
+          ['OS != "zos"', {
+            'cflags': [ '-O0', '-fwrapv' ]
+          }],